packages/replayio/src/utils/installation/installLatestRelease.ts

```diff
@@ -150,18 +149,27 @@ async function downloadReplayFile({ onRetry }: { onRetry?: (attempt: number) =>
   throw new Error("Download failed, giving up");
 }
 
-async function extractBrowserArchive(runtimeBaseDir: string, downloadFilePath: string) {
+function extractBrowserArchive(runtimeBaseDir: string, downloadFilePath: string) {
   logger.info(`ExtractBrowserArchive:Extracting`, { downloadFilePath });
 
   const tarResult = spawnSync("tar", ["xf", runtimeMetadata.downloadFileName], {
     cwd: runtimeBaseDir,
   });
   if (tarResult.status !== 0) {
+    const hasXz = spawnSync("which", ["xz"]).status === 0;
+
     logger.error("ExtractBrowserArchive:Failed", {
       downloadFilePath,
+      hasXz,
       stderr: String(tarResult.stderr),
     });
 
-    throw new Error("Unable to extract browser archive");
+    let message = "Unable to extract browser archive";
+
+    if (!hasXz) {
```

Comment:
it's tempting to suggest some concrete installation instructions for this but it might differ based on the linux distro used and other factors